tests/integration/tier1/remove-excluded-pkgs/main.fmf

```diff
@@ -1,10 +1,6 @@
 summary: remove-excluded-pkgs
 
 tier: 0
-adjust:
-  enabled: false
-  when: >
-    distro != centos-8
 
 /good:
```

Comment:
Can we remove this sub test, since there is only one test case?

Reply:
Good catch, I removed it.